applications/launchpad/gui-react/src/containers/BaseNodeContainer/StateTag.tsx

```diff
@@ -0,0 +1,30 @@
+import { useAppSelector } from '../../store/hooks'
+
+import Tag from '../../components/Tag'
+import { selectRunning, selectHealthy } from '../../store/baseNode/selectors'
+import t from '../../locales'
+
+const StateTag = () => {
+  const running = useAppSelector(selectRunning)
+  const healthy = useAppSelector(selectHealthy)
+
+  if (!running) {
+    return null
+  }
+
+  if (!healthy) {
+    return (
+      <Tag variant='small' type='warning'>
+        {t.common.adjectives.unhealthy}
```

Comment:
`Unhealthy` - not sure how this word sounds and can be perceived by the person without the _IT_ background, and some launchpad users may not be have such experience. I think that's minor, and leaving this comment to share my thought rather than to suggest any change here.

Reply:
thank you, we'll probably workshop this on Monday anyway